src/style.css

```diff
@@ -1,3 +1,91 @@
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  --main-color: #00acdc;
+}
+/*In general we should use "!important" very carefully.
+But in case of hidden element it useful*/
+.visually-hidden {
+  position: absolute !important;
+  width: 1px !important;
+  height: 1px !important;
+  margin: -1px !important;
+  border: 0 !important;
+  padding: 0 !important;
+  overflow: hidden !important;
+  clip: rect(0,0,0,0) !important;
+  white-space: nowrap !important;
+}
```

Comment:
if you want to achieve this, you can just:
```css
  visibility: 'hidden';
  position: 'absolute';
```

Reply:
but if I use "visibility: 'hidden';" element will disappeared for searching and screen readers, no?

Reply:
so then you can just:
```
  position: absolute;
  left:-9999px;
```